templates/result.html

```diff
@@ -3,7 +3,5 @@
 {% block content %}
     <h1>{% block title %} Results for {{ result['title'] }} {% endblock %}</h1>
 
-    <p>
-        {{ result['text'] }}
-    </p>
+    <p> {{ result['text'] }} </p>
```

Comment:
I agree with the change but will likely be overwritten or cause merge conflicts with PRs being made at the same time.
Perhaps we shouldn't make changes linked to different issues in a single PR (i.e. only touch tests in a PR about a test issue)?

Reply:
Reverted to the previous version.